iwyu_preprocessor.cc

```diff
@@ -209,6 +217,19 @@ void IwyuPreprocessorInfo::HandlePragmaComment(SourceRange comment_range) {
     return;
   }
 
+  if (HasOpenBeginKeep(this_file_entry)) {
+    if (MatchOneToken(tokens, "end_keep", 1, begin_loc)) {
+      ERRSYM(this_file_entry) << "end_keep pragma seen\n";
+      SourceLocation temp = begin_keep_location_stack_.top();
+	  begin_keep_location_stack_.pop();
+	  keep_location_ranges_[GetFileEntry(temp)].push_back(clang::SourceRange{temp,begin_loc});
```

Comment:
These two lines are overindented. And the second might possibly be too long (column limit is 80)

Reply:
Also we always have space after comma (`clang::SourceRange(temp, begin_loc)`).

Further, would be nice if you added a `using clang::SourceRange` to the top, so we don't need to qualify it here.